### PR TITLE
Issue #3171785 by agami4: Make help text search button not visible for the accessibility tree

### PR DESCRIPTION
**Pull request by agami4**
## Problem
The search icon in the menu bar contains a help text. However, this help text isn't actually visible for screen readers.

## Solution
Replace <span class="invisible">Search</span> with <span class="sr-only"Search</span> and add `aria-hidden="true"`

## Issue tracker
https://www.drupal.org/project/social/issues/3171785

## How to test
*For example*
- [ ] Open inspector element and check new help text search button

## Screenshots
search-button-help-text

## Release notes
The help text search button not visible for the accessibility tree.

## Change Record
-

## Translations
-

### CODE REVIEW

File: themes/socialbase/templates/block/block--system-menu-block--main.html.twig [hunk 1]
```diff
@@ -49,7 +49,7 @@
   {% if content.links.search_block is defined %}
     <div class="navbar__open-search-control">
       <a href="{{ '/search/all' }}" class="navbar__open-search-block" rel="search" title="{{ 'Click to open search box'|t }}">
-        <span class="invisible">{{ 'Search'|t }}</span>
+        <span class = "sr-only" aria-hidden="true">{{ 'Search'|t }}</span>
```
Comment (Kingdutch) [inline themes/socialbase/templates/block/block--system-menu-block--main.html.twig:52]: @agami4 Why did you add the `aria-hidden="true"`? This way it's still not visible to screenreaders.

**Review by Kingdutch — CHANGES_REQUESTED** [review]

File: themes/socialbase/templates/block/block--system-menu-block--main.html.twig (hunk 1 above)
Comment (agami4) [reply themes/socialbase/templates/block/block--system-menu-block--main.html.twig]: @Kingdutch, I've removed the `aria-hidden` attribute and checked via 'Voice Over'

**Review by Kingdutch — APPROVED** [review]
